.github/dependabot.yml

```diff
@@ -2,12 +2,15 @@ version: 2
 updates:
   # Enable version updates for npm
   - package-ecosystem: "npm"
+    reviewers:
+      - "microsoft/team-jason"
+      - "microsoft/team-rohan"
```

Comment:
Since team-jason doesn't exist and use yet unclear, if you want to commit now, skip this.